DigitalOffice/Tests/TimeManagementServiceTests/Repositories/WorkTimeRepositoryTests.cs

```diff
@@ -143,5 +143,57 @@ public void CorrectlyReturnsWorkTime()
                 workTimesOfWorker2.Count);
         }
         #endregion
+
+        #region ChangeWorkTimeTests
+        [Test]
+        public void ShouldChangeWorkTimeWhenDataIsCorrect()
+        {
+            var id = Guid.NewGuid();
+
+            var oldTime = new DbWorkTime
+            {
+                Id = id,
+                WorkerUserId = Guid.NewGuid(),
+                StartTime = new DateTime(2020, 1, 1, 1, 1, 1),
+                EndTime = new DateTime(2020, 2, 2, 2, 2, 2),
+                Title = "Example",
+                ProjectId = Guid.NewGuid(),
+                Description = "ExampleDescription"
+            };
+
+            var newTime = new DbWorkTime
+            {
+                Id = id,
+                WorkerUserId = Guid.NewGuid(),
+                StartTime = new DateTime(2020, 1, 1, 1, 1, 1),
+                EndTime = new DateTime(2020, 2, 2, 2, 2, 2),
+                Title = "ExampleTitle",
+                ProjectId = Guid.NewGuid(),
+                Description = "Example"
+            };
+
+            dbContext.WorkTimes.Add(oldTime);
+            dbContext.SaveChanges();
+
+            Assert.True(repository.ChangeWorkTime(newTime));
```

Comment:
Add verifying that entity in db was changed

Reply:
fixed